todo/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Task(models.Model):
+    #author = models.ForeignKey('auth.User')
+    title = models.TextField()
```

Comment:
Ignore my [last comment](https://github.com/dandel10n/todo-list/pull/14#issuecomment-223588017).

We don't need `title` to be `TextField`. I think we can limit `title` with 255 symbols.